python/lsst/ts/ofc/state_estimator.py

```diff
@@ -135,6 +136,14 @@ def dof_state(
 
         # Compute the pseudo-inverse of the sensitivity matrix
         # rcond sets the truncation of different modes.
+        # If rcond is None, it is computed from the singular values
+        # of the sensitivity matrix, using the truncation index as reference.
+        if self.rcond is None:
+            if self.truncate_index is None:
+                raise ValueError("Truncation index not set in the controller.")
```

Comment:
I think this should mention that they are both `None` instead of assuming which one the user wants to set.